src/kv/RocksDBStore.cc

```diff
@@ -1436,6 +1436,29 @@ void RocksDBStore::get_statistics(Formatter *f)
       f->close_section();
     }
   }
+
+  if (cct->_conf->rocksdb_collect_all_cf_compaction_stats) {
```

Comment:
IMO that's an overkill to have a separate config switch for this stuff. May be just pring all the column handles under "if (rocksdb_collect_comapction_stats)" ?

Reply:
Thanks for your suggestion. Indeed, putting it in rocksdb_collect_comapction_stats is already OK.

Reply:
My initial concern was that someone might call this interface in our project.